tests/docker/README.md

```diff
@@ -26,3 +26,16 @@ then install the composer dependencies and run tests:
 $ composer install
 $ ./vendor/bin/phpunit
 ```
+
+<br />
+
+Docker and Composer packages 
+======================
+
+If you want to have a full testing environment with composer packages installed:
+
+```shell
+$ ./start.sh
```

Comment:
I would rather add `docker-compose` commands here, instead of a script

Reply:
My goal is to speed up as much as possible time to start testing, so I removed all commands.